PHP/CodingConventions.md

```diff
@@ -86,7 +93,8 @@ if ( F::App()->wg->User->can( 'edit' ) && F::App()->wg->Skin->getSkinName() == '
 }
 
 // Good - each check split onto its own line
-if ( F::app()->wg->User->isAllowed( 'edit' ) &&
+if ( 
+	F::app()->wg->User->isAllowed( 'edit' ) &&
```

Comment:
The previous version looked so bad, I never used it. Thanks for making it officialy unsupported!